src/app/index.html

```diff
@@ -90,6 +92,7 @@
       window.addEventListener("load", function() {
 
         System.import('app/boot').then(null, console.error.bind(console));
+        System.import('dompurify').then(null, console.error.bind(console));
```

Comment:
I don't understand this line. We already
```typescript
import * as DomPurify from "dompurify";
```
in `clarity-icons-api.ts`, why do we need to manually import DomPurify from the page itself? If we actually _need_ it, then this is an issue because our users would need it too.